ProjectEureka/project-eureka-flutter
# PE-77 | Implemented the Complete UI for MoreDetailsPage & ChooseBestAnswer

**What I Did:**
- Extracted some Widgets from the MoreDetailsPage created by @dominikyz and made it a reusable component. 
- Completed the UI for MoreDetailsPage & ChooseBestAnswer according to Figma.
- The only thing that's not working yet is the Chat Buttons since I'm still waiting for another Branch to Merge into Main.

**How to Test:**
1. From the HomeScreen, click the "More Details" button.
2. You should then be redirected to a new screen. 
3. Play around with this screen and try to break it.
